samples/sdl_ttf/Makefile

```diff
@@ -0,0 +1,12 @@
+DEBUG = y
+XBE_TITLE = sdl
+SRCS += $(wildcard $(CURDIR)/*.c)
```

Comment:
Avoid wildcards

Reply:
This might be the cause of the following build error I receive in msys.

```[ LD       ] main.exe
make: execvp: lld-link: Argument list too long
make: *** [/home/Mike/projects/nxdk/samples/sdl_ttf/../../Makefile:114: main.exe] Error 127```

Reply:
No, it's (almost certainly) because we list all source files, instead of moving larger groups of them into libraries.
SDL_ttf added a couple of more, and now you are hitting limits (which are very low on Windows).

I'm actually surprised this worked for as long as it did.

You can temporarily work around it by using shorter folder-names.

I suggest you create a separate issue for this problem, as it's not specific to this PR.